[rospy.client][INFO] 2020-12-25 06:45:02,426: init_node, name[/train_3388_1608878702398], pid[3388]
[xmlrpc][INFO] 2020-12-25 06:45:02,427: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-12-25 06:45:02,427: Started XML-RPC server [http://192.168.0.191:42745/]
[rospy.init][INFO] 2020-12-25 06:45:02,427: ROS Slave URI: [http://192.168.0.191:42745/]
[rospy.impl.masterslave][INFO] 2020-12-25 06:45:02,428: _ready: http://192.168.0.191:42745/
[rospy.registration][INFO] 2020-12-25 06:45:02,429: Registering with master node http://192.168.0.201:11311
[xmlrpc][INFO] 2020-12-25 06:45:02,429: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-12-25 06:45:02,528: registered with master
[rospy.rosout][INFO] 2020-12-25 06:45:02,528: initializing /rosout core topic
[rospy.rosout][INFO] 2020-12-25 06:45:02,533: connected to core topic /rosout
[rospy.simtime][INFO] 2020-12-25 06:45:02,535: initializing /clock core topic
[rospy.simtime][INFO] 2020-12-25 06:45:02,537: connected to core topic /clock
[rospy.internal][INFO] 2020-12-25 06:45:02,541: topic[/clock] adding connection to [http://192.168.0.201:42071/], count 0
[rospy.internal][INFO] 2020-12-25 06:45:02,549: topic[/bev/temporal_bev_image] adding connection to [http://192.168.0.201:36431/], count 0
[rospy.internal][INFO] 2020-12-25 06:45:02,564: topic[/is_start_episode] adding connection to [http://192.168.0.201:46789/], count 0
[rospy.internal][INFO] 2020-12-25 06:45:02,565: topic[/odom] adding connection to [http://192.168.0.201:42071/], count 0
[rospy.internal][INFO] 2020-12-25 06:45:02,570: topic[/scan] adding connection to [http://192.168.0.201:34643/], count 0
[rospy.internal][INFO] 2020-12-25 06:45:02,571: topic[/scan] adding connection to [http://192.168.0.201:42071/], count 0
[rospy.internal][INFO] 2020-12-25 06:45:02,572: topic[/start_goal_points] adding connection to [http://192.168.0.201:40143/], count 0
[rospy.internal][INFO] 2020-12-25 06:45:02,708: topic[/episode_watcher/is_finish_episode] adding connection to [/episode_manager], count 0
[rospy.internal][INFO] 2020-12-25 06:45:02,805: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-12-25 06:45:02,806: topic[/cmd_vel] adding connection to [/gazebo], count 0
[rospy.core][INFO] 2020-12-25 06:45:10,505: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-12-25 06:45:10,511: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-12-25 06:45:10,512: topic[/cmd_vel] removing connection to /gazebo
[rospy.internal][INFO] 2020-12-25 06:45:10,512: topic[/episode_watcher/is_finish_episode] removing connection to /episode_manager
[rospy.internal][INFO] 2020-12-25 06:45:10,512: topic[/clock] removing connection to http://192.168.0.201:42071/
[rospy.internal][INFO] 2020-12-25 06:45:10,513: topic[/bev/temporal_bev_image] removing connection to http://192.168.0.201:36431/
[rospy.internal][INFO] 2020-12-25 06:45:10,513: topic[/odom] removing connection to http://192.168.0.201:42071/
[rospy.internal][INFO] 2020-12-25 06:45:10,514: topic[/start_goal_points] removing connection to http://192.168.0.201:40143/
[rospy.internal][INFO] 2020-12-25 06:45:10,514: topic[/scan] removing connection to http://192.168.0.201:34643/
[rospy.internal][INFO] 2020-12-25 06:45:10,515: topic[/scan] removing connection to http://192.168.0.201:42071/
[rospy.internal][INFO] 2020-12-25 06:45:10,516: topic[/is_start_episode] removing connection to http://192.168.0.201:46789/
[rospy.impl.masterslave][INFO] 2020-12-25 06:45:10,516: signal-2
[rospy.internal][WARNING] 2020-12-25 06:45:10,517: Unknown error initiating TCP/IP socket to 192.168.0.201:47393 (http://192.168.0.201:40143/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.internal][WARNING] 2020-12-25 06:45:10,518: Unknown error initiating TCP/IP socket to 192.168.0.201:38887 (http://192.168.0.201:42071/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.internal][WARNING] 2020-12-25 06:45:10,519: Unknown error initiating TCP/IP socket to 192.168.0.201:50295 (http://192.168.0.201:46789/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 563, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[rospy.core][INFO] 2020-12-25 06:45:11,006: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-12-25 06:45:11,035: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-12-25 06:45:11,065: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-12-25 06:45:11,096: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-12-25 06:45:11,110: signal_shutdown [atexit]
